pkg/engine/template_generator_test.go

```diff
@@ -9,9 +9,8 @@ import (
 	"testing"
 	"text/template"
 
-	"github.com/pkg/errors"
-
 	"github.com/Azure/aks-engine/pkg/api"
+	"github.com/pkg/errors"
```

Comment:
FYI, some linters will complain about the imports' order change. Not sure if AKSe's does.

Reply:
My IDE probably did that. It passes the linter though.